### PR TITLE
Update the README for Release Candidate

### PR DESCRIPTION
**Pull request by steveluscher**

### CODE REVIEW

**Comment by steveluscher** [comment]
* **#3003** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/3003?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @steveluscher and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by lorisleiva — APPROVED** [review]
Looks good! I had a quick look at the codec/Kinobi section and I don't think anything else needs updating.

File: packages/library/README.md [hunk 1]
```diff
@@ -205,9 +180,9 @@ const rpc = createSolanaRpc('http://127.0.0.1:8899');
 const slot = await rpc.getSlot().send();
 ```
 
-### Using Custom RPC Transports
+### Custom RPC Transports
 
-The `createSolanaRpc` function communicates with the RPC server using a default HTTP transport that should satisfy most use cases. However, you may provide your own transport or decorate existing ones to communicate with RPC servers in any way you see fit. In the example below, we explicitly create a transport and use it to create a new RPC client via the `createSolanaRpcFromTransport` function.
+The `createSolanaRpc` function communicates with the RPC server using a default HTTP transport that should satisfy most use cases. You can provide your own transport or decorate an existing one to communicate with RPC servers in any way you see fit. In the example below, we explicitly create a transport and use it to create a new RPC client via the `createSolanaRpcFromTransport` function.
```
Comment (mcintyre94) [inline packages/library/README.md:185]: Nitpick: I'm not sure quite what "decorate an existing one" means, given we don't have decorators. I wonder if "wrap an existing one" would be a bit clearer.

File: packages/library/README.md [hunk 2]
```diff
@@ -401,7 +372,7 @@ const rpc = createRpc({ api, transport });
 
 Note that the `createSolanaRpcApi` function is a wrapper on top of the `createRpcApi` function which adds some Solana-specific transformers such as setting a default commitment on all methods or throwing an error when an integer overflow is detected.
 
-#### Creating your Own API Methods
+#### Creating Your Own API Methods
 
 The new library’s RPC specification supports an _infinite_ number of JSON-RPC methods with **zero increase** in bundle size.
```
Comment (mcintyre94) [inline packages/library/README.md:377]: Nitpick: The unchanged bit below this refers to a custom Quicknode method for Ethereum. Their [Solana docs](https://www.quicknode.com/docs/solana) don't have any equivalent custom methods AFAICT. Might be worth switching out for a Helius link, eg https://docs.helius.dev/compression-and-das-api/digital-asset-standard-das-api/get-asset?

**Review by mcintyre94 — APPROVED** [review]
LGTM! A few small nitpicky suggestions but this looks really great IMO.

**Pull request merged**